test/metabase/api/database_test.clj

```diff
@@ -1370,26 +1372,69 @@
                      (:details d)
                      (select-keys d [:password-source :password-value]))))))))
 
+;; these descriptions use deferred-tru because the `defsetting` macro complains if they're not, but since these are in
+;; tests they won't get scraped for i18n purposes so it's ok.
```

Comment:
hmm, I feel like we should just tweak `defsetting` to not complain about this if we're in a test namespace. I'll open a tech debt issue for that

Reply:
never mind, this was easy enough to do so I just put up https://github.com/metabase/metabase/pull/28170

Reply:
Thank you!